addons/mrp/tests/test_bom.py

```diff
@@ -599,6 +599,15 @@ def test_20_bom_report(self):
             'time_stop': 15
         })
 
+        capacity_cheese_cake = self.env['mrp.workcenter.capacity'].create({
+            'product_id': cheese_cake.id,
+            'workcenter_id': workcenter_2.id,
+            'time_start': 2,
+            'time_stop': 1,
+        })
+
+        workcenter_2.write({'capacity_ids': [(4, capacity_cheese_cake.id)]})
```

Comment:
This line:

`workcenter_2.write({'capacity_ids': [(4, capacity_cheese_cake.id)]})`

is redundant with the `'workcenter_id': workcenter_2.id,` in the `create`